mem-12-q.xml

```diff
@@ -11671,25 +11671,27 @@ There is no one way to say "firefighter". Some options are {qul SuvwI':n} "fire
       <column name="notes_ru"></column>
       <column name="notes_zh_HK"></column>
       <column name="notes_pt"></column>
-      <column name="notes_fi"></column>
+      <column name="notes_fi">Englanninkielinen määritelmä "government" tarkoittaa yleensä hallitusta, mutta yhdysvaltojen englannissa laajemmin valtiota kokonaisuudessa. Esimerkki varmistaa, että klingonin sanassa on kyse jälkimmäisestä merkityksestä.</column>
       <column name="hidden_notes"></column>
       <column name="components"></column>
-      <column name="examples"></column>
+      <column name="examples">
+▶ {qum DuSaQ:n} "public school"</column>
```

Comment:
When an example has a linked entry, it's better not to provide another translation here as the two can get out of sync if one of them changes. (There are actually probably a couple of cases like this in the database.) 

I think the right thing to do is to copy the translation from the entry in code and automatically display it. (One day I will redesign the database and rewrite the code to make this easier.)